### PR TITLE
Feature: Created Information Component

### PR DESCRIPTION
**Pull request by camitaylor**
# [Feature: Created Information Component](https://trello.com/c/zKmzONTO/7-create-information-page)

## A short description of what this PR does. 
As a user, I would like to see a page, so I can get information about the hacktoberfest event.

## How was it tested?
This was linted with Yarn and tested with Jest.

### Yarn Test
Screen Shot 2021-07-19 at 5 10 50 PM

### Yarn Lint
Screen Shot 2021-07-19 at 3 50 30 PM

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/aaronshivers/snapit-hacktoberfest/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
```
Comment (dylhack) [inline src/main.jsx:3]: You may want to keep an index.css for styling later, but for now this is fine.

**Review by aaronshivers — CHANGES_REQUESTED** [review]
⛏️  The PR title does not follow the correct format. Please prefix it with `Feature: `

File: src/pages/Information/Information.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+const Information = () => (
+  <>
+    <div data-test="information-display">
+      <h2>What is HacktoberFest?</h2>
```
Comment (aaronshivers) [inline src/pages/Information/Information.jsx:6]: ♻️  Could you add tests for these headings?

**Review by aaronshivers — CHANGES_REQUESTED** [review]

File: src/pages/Information/Information.test.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  test("renders information display", () => {
+    const wrapper = setup();
+    const informationDisplay = findByTestAttribute(
+      wrapper,
+      "information-display"
+    );
+    expect(informationDisplay.length).toBe(1);
+  });
+});
```
Comment (aaronshivers) [inline src/pages/Information/Information.test.jsx:24]: 😎 Nice!

**Comment by dylhack** [comment]
I'll submit my review after @aaronshivers

File: src/pages/Information/Information.jsx (hunk 2 above)
Comment (camitaylor) [reply src/pages/Information/Information.jsx]: Modified the title and added tests for each tag.

File: src/main.jsx (hunk 1 above)
Comment (camitaylor) [reply src/main.jsx:3]: Aaron wanted that file removed.

Comment (aaronshivers) [reply src/main.jsx:3]: Yeah, I should have removed these auto-generated files earlier.

**Review by aaronshivers — APPROVED** [review]

File: src/pages/Information/index.js [hunk 4]
```diff
@@ -0,0 +1 @@
+export { default } from "./Information";
```
Comment (dylhack) [inline src/pages/Information/index.js:1]: Couldn't Information.jsx be written in this index since it's just double exporting at this point?

Comment (camitaylor) [reply src/pages/Information/index.js:1]: The export file was Aaron's suggestion on my file structure.

**Review by dylhack — APPROVED** [review]
LGTM 👍

**Pull request merged**